
float32[] previousScan
float32[] previousScan2
float32[] previousScan3
float32[] currentScan


float32[] velocityModule

############################################################
# float32[] velocityCenter # only needed for polar coordinates
############################################################